### ISSUE TITLE
`var_export` return type

### ISSUE THREAD

**Issue by weirdan** [vimeo/psalm#636]
`var_export()` is documented as returning `string` when its second parameter is `true`, and returning `null` otherwise. Is there a way to specify it in CallMap, with two signatures possibly? I'm getting 'Cannot concatenate with a possibly null string|null', and it's a bit annoying to suppress it all over the place.

```php
<?php
"" . var_export([], true); // should not complain
"" . var_export([], false); // should complain
"" . var_export([]); // and here as well
```
